conf/application.prod.conf

```diff
@@ -9,10 +9,10 @@ silhouette {
   oauth1TokenSecretProvider.secureCookie=true
 
   # Facebook provider
-  facebook.redirectURL="https://play-silhouette-angular-seed.herokuapp.com/"
+  facebook.redirectURL="https://play-silhouette-angular-seed.herokuapp.com/authenticate/facebook"
 
   # Google provider
-  google.redirectURL="https://play-silhouette-angular-seed.herokuapp.com"
+  google.redirectURL="https://play-silhouette-angular-seed.herokuapp.com/authenticate/google"
```

Comment:
Must be `google.redirectURL="https://play-silhouette-angular-seed.herokuapp.com"`